source/PlayState.hx

```diff
@@ -3908,6 +3908,20 @@ class PlayState extends MusicBeatState
 		}
 	}
 	#end
+	
+	public var screenGoesBlack:Bool = true;
+	function blackScreen(?screenGoesBlack:Bool = true) { // this can also be used for winter horrorland
```

Comment:
There is no reason to declare both a public field and function parameter with the same name here.

Reply:
I know, I wanted to be careful so the engine wouldn’t get confused